packages/web/src/scss/components/TextArea/index.html

```diff
@@ -19,13 +19,24 @@
     <textarea id="textareaFilledMessage" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
     <div class="TextArea__message">Message</div>
   </div>
+  <div class="TextArea TextArea--danger">
+    <label for="textareaDanger" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaDanger" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+  </div>
+  <!-- @deprecated Will be removed in next major version. -->
   <div class="TextArea TextArea--error">
     <label for="textareaError" class="TextArea__label TextArea__label--required">Label</label>
     <textarea id="textareaError" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
   </div>
+  <div class="TextArea TextArea--danger">
+    <label for="textareaDangerMessage" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaDangerMessage" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <!-- @deprecated Will be removed in next major version. -->
   <div class="TextArea TextArea--error">
-    <label for="textareaErrorMessage" class="TextArea__label TextArea__label--required">Label</label>
-    <textarea id="textareaErrorMessage" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <label for="textareaErrorMessage_2" class="TextArea__label TextArea__label--required">Label</label>
```

Comment:
is there textareaErrorMessage? (without the suffix _2)

Reply:
Yes, of course